cli/deno_dir.rs

```diff
@@ -634,119 +677,76 @@ fn fetch_remote_source(
   ))
 }
 
-fn get_cache_filename(basedir: &Path, url: &Url) -> PathBuf {
```

Comment:
Can you move this one too?

Reply:
@ry I should have moved all of these just moments ago